interface-definitions/include/firewall/match-ether-type.xml.i

```diff
@@ -0,0 +1,34 @@
+<!-- include start from firewall/match-ether-type.xml.i -->
+<leafNode name="ethernet-type">
+  <properties>
+    <help>Ethernet type</help>
+    <completionHelp>
+      <list>8021q 8021ad arp ip ip6 vlan</list>
+    </completionHelp>
+    <valueHelp>
+      <format>8021q</format>
```

Comment:
Probably should use `802.1q` and `802.1ad` to be inline with the standards

Reply:
I know it might need transcoding in the Python scripts but the CLI exported to the user looks more consistent.